### PR TITLE
Revert "Added Cold Brew Recipe and Eggs Recipe Name"

### PR DESCRIPTION
**Pull request by boncobee**
## 🎯 Objective

Accidentally merged without having @jzofficial66 review it first (apologies)

## 🛠️ Changes made

_Note what you actual changes you're proposing_ 

## 👥 Who needs to review this Pull Request? 

_@mention some teammates to review your work._
_Pull Requests should include commits from each member of your group._

### CODE REVIEW

File: breakfast/readme.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Eggs a la Scramble 
+# Replace with recipe name
 
 ## Servings: 2
```
Comment (jzofficial66) [inline breakfast/readme.md:3]: Use < br > for line breaks. ## for headings

**Review by dhruvkc — APPROVED** [review]
changes look good to me! @wyattmdavis feel free to review and close out!

**Review by wyattmdavis — COMMENTED** [review]
@boncobee can you take a look at the code conflict? I would, but I'm having a bit of trouble following how this request fits in with our other branches!

**Comment by jzofficial66** [comment]
Hey all. This is an accidental request to revert a previous approval. Going to close.

**Pull request closed without merging**